config/config.go

```diff
@@ -47,15 +50,16 @@ type Config struct {
 
 func FromFlags() (*Config, error) {
 	cfg := &Config{}
-	var network, coinbase, gas, coa, key string
+	var evmNetwork, coinbase, gas, coa, key string
 
 	// parse from flags
 	flag.StringVar(&cfg.DatabaseDir, "database-dir", "./db", "path to the directory for the database")
-	flag.StringVar(&cfg.RPCHost, "rpc-host", "localhost", "host for the RPC API server")
+	flag.StringVar(&cfg.RPCHost, "rpc-host", "", "host for the RPC API server")
 	flag.IntVar(&cfg.RPCPort, "rpc-port", 3000, "port for the RPC API server")
```

Comment:
I feel that we should keep `8545` here, just like Geth. Some tools expect to find a JSON-RPC server on `localhost:8545`. Just to keep parity.